### ISSUE TITLE
If I flash will it continue to work in windows/osx/etc?

### ISSUE THREAD

**Issue by fabriciomrtnz** [qca/open-ath9k-htc-firmware#84]
Hello,

Im not clear if I flash the device (alfa network) will it continue to work on other OSs.

-Thanks

**Comment by olerem** [comment]
What do you mean with flash? Solder an spi-flash and boot firmware?

**Comment by fabriciomrtnz** [comment]
If I understand right, this code overwrites the firmware on the device. So if I overwrite what ever is there from factory. Will my device continue to work in other OSs or will it only work in linux with the ath9k_htc driver?

**Comment by olerem** [comment]
no, this firmware is loaded to RAM of this adapter and lost after poweroff. So the answer is no, it is not affecting any OS

**Comment by fabriciomrtnz** [comment]
Ok, thank you.
